allennlp/training/trainer_pieces.py

```diff
@@ -54,6 +54,7 @@ def from_params(
                     "are ignoring them, using instead the loaded model parameters."
                 )
 
+            # TODO(mattg): This should be updated now that directory_path no longer exists.
```

Comment:
This whole class is getting removed in #3625

Reply:
#3627 :smile:

Reply:
Thanks for the reminder. That should very robustly take care of this TODO! ;)